draft-ietf-tls-esni.md

```diff
@@ -416,9 +416,21 @@ OuterExtensions consists of one or more ExtensionType values, each of which
 reference an extension in ClientHelloOuter.
 
 When sending ClientHello, the client first computes ClientHelloInner, including
-any PSK binders. It then computes a new value, the EncodedClientHelloInner, by
-first making a copy of ClientHelloInner. It then replaces the
-legacy\_session\_id field with an empty string.
+any PSK binders. It then computes a new value, the EncodedClientHelloInner,
+which is the following structure:
+
+~~~
+    struct {
+        ClientHello client_hello;
+        opaque padding[N];
+    } EncodedClientHelloInner;
+~~~
+
+The `client_hello` field is computed by first making a copy of
+ClientHelloInner with the `legacy_session_id` field with an empty string.
+Note this field uses the ClientHello structure, defined in Section 4.1.2
+of {{RFC8446}} which does not include the Handshake structure's four byte
```

Comment:
```suggestion
Note this field uses the ClientHello structure, defined in {{Section 4.1.2
of RFC8446}} which does not include the Handshake structure's four byte
```

Reply:
YOU CAN DO THAT?!?!!? Wow, I need to go update all my documents now.